lib/topological_inventory/ansible_tower/operations/core/ansible_tower_client.rb

```diff
@@ -0,0 +1,129 @@
+require "topological_inventory/ansible_tower/logging"
+require "topological_inventory/ansible_tower/connection"
+require "topological_inventory/ansible_tower/operations/core/sources_api_client"
+require "topological_inventory/ansible_tower/operations/core/topology_api_client"
+
+module TopologicalInventory
+  module AnsibleTower
+    module Operations
+      module Core
+        class AnsibleTowerClient
```

Comment:
Calling this AnsibleTowerClient is a little confusing and most of this looks like it can just live with the order code